mobile-app/src/atoms/atoms.ts

```diff
@@ -582,3 +582,5 @@ export const settingAtom = atomFamily((settingKey: string) =>
     { getOnInit: true },
   ),
 );
+
+export const profileFormDataAtom = atom({ displayName: "", username: "" }); //maybe change to use user atom data?
```

Comment:
ew gross, make the default value the same as `currentUserAtom` except the id and type it with `userT` (like u did in that other place)